src/test/java/org/killbill/billing/plugin/adyen/api/TestAdyenPaymentPluginHttpErrors.java

```diff
@@ -131,13 +131,13 @@ public void testAuthorizeAdyenNotReachable() throws Exception {
         final PaymentTransactionInfoPlugin result = authorizeCall(account, payment, callContext, pluginApi, creditCardPaymentProperties());
         assertEquals(result.getStatus(), PaymentPluginStatus.CANCELED);
         assertEquals(result.getGatewayError(), "nothing-here.to");
-        assertEquals(result.getGatewayErrorCode(), "java.net.UnknownHostException");
```

Comment:
The exception doesn't show up in the `Error`? I think this loss of information would be an issue for debugging outages.

Reply:
I was thinking maybe it is more appropriate to put both [`Exception Class`](https://github.com/killbill/killbill-adyen-plugin/blob/adyen-plugin-0.5.5/src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java#L209) and the [`Exception Message`](https://github.com/killbill/killbill-adyen-plugin/blob/adyen-plugin-0.5.5/src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java#L179) to  `getGatewayError` since the exception class isn't actually a code either. How does that sound?

Reply:
That sounds good! What format are you thinking?

One way would be to use JSON (similar to https://github.com/killbill/killbill-plugin-framework-ruby/blob/4681bfb8caafda0b3192ce368cb2e2e12cb17659/lib/killbill/helpers/active_merchant/gateway.rb#L141), but it's not user friendly (Kaui display).

Alternatively, we could just return something like `ExceptionClass: exceptionMessage` (e.g. `java.net.UnknownHostException: nothing-here.to`) and have something more structured in the plugin properties?

Reply:
Done in 7d3b3c0